src/utils/octet.hpp

```diff
@@ -25,6 +25,11 @@ struct octet
     {
     }
 
+    inline uint8_t getValue() const
```

Comment:
Please remove this function.

You can directly cast to uint8_t in order to convert octet to uint8_t

uint8_t aaa = (uint8_t)bbb;

Reply:
Sure, will do this.

Reply:
`octet` has a special converter for `uint8_t`. Please directly use this converter, and remove the getValue().

Example:

```
octet someOctet = ....
uint8_t someUint8 = (uint8_t)someOctet 
```